pymeasure/instruments/hp/hp663X2A.py

```diff
@@ -26,8 +26,8 @@
 from pymeasure.instruments.validators import strict_discrete_set
 
 
-class HP66312A(SCPIMixin, Instrument):
-    """ Represents the HP / Agilent / Keysight 66312AA Dynamic Measurement DC Source and
+class HP663X2A(SCPIMixin, Instrument):
+    """ Represents the HP / Agilent / Keysight 66312A/66332A Dynamic Measurement DC Source and
```

Comment:
Better to rename to the docstring to "663X2A series", unless there are only these two devices.